src/pages/collators/columns.tsx

```diff
@@ -28,6 +28,7 @@ const getAmountDelegated = (candidate: ParachainStakingCandidate, address: strin
 export const nameColumn: ColumnDef<TCollator> = {
   header: 'Collator',
   accessorKey: 'collator',
+  accessorFn: ({ identityInfo }) => identityInfo?.display || '0',
```

Comment:
Maybe we should replace the `0` with the actual text that you display in case it is empty? This is just nitpicking but it's a bit weird that the 'Unknown' collators seem to not follow the alphabetic ordering 

```suggestion
  accessorFn: ({ identityInfo }) => identityInfo?.display || 'Unknown',
```

Reply:
oh, maybe this is not too intuitive. I prefer to have all the meaningful names together when ordering (from A to Z) and then Unkwnon as a special category. Since it is an arbitrary word, its order doesn't matter.